### ISSUE TITLE
Deleting synset "Ritz"

### ISSUE THREAD

**Issue by stobbsobe** [globalwordnet/english-wordnet#986]
**Synset**
Ritz at https://en-word.net/lemma/Ritz
oewn-04102203-n (Interlingual Index: i58199)

**Motivation**

1. Redundant as already covered in https://en-word.net/lemma/ritz.
2. Meaning is more accurately "the actual Ritz luxury hotels created by the the Swiss hotelier César Ritz (1850–1918)." (See https://www.collinsdictionary.com/dictionary/english/ritz, and https://www.oxfordreference.com/display/10.1093/acref/9780199543793.001.0001/acref-9780199543793-e-2104#:~:text=Source%3A%20The%20Oxford%20Dictionary%20of,show%20of%20luxury%20or%20extravagance)

Please let me know how I can further explain if needed.

**Comment by jmccrae** [comment]
I agree, the "the ritz of something" is better covered by the other sense and we would not try to cover every famous hotel name.

**Comment by stobbsobe** [comment]
Hi jmccrae, 

Please could we delete the definition of Ritz as 'an ostentatiously elegant hotel' in that case? 

Please let me know if we can provide any further explanation.

**Comment by jmccrae** [comment]
My slight concern would be that this change may break some annotation. For example in the sentence 

> He stayed at the Ritz

If we change the tagging of 'Ritz' from 'an ostentatiously elegant hotel' to 'ostentatious display of elegance' I worry that this might be a change of meaning. 

I might alternatively suggest that we change the definition of 'Ritz' to 'a hotel that is part of the chain of hotels founded by Cesar Ritz'. 

While this synset would not be of sufficient interest to be included in OEWN as a novel synset, this change would also have less risk of breaking other's annotations.

**Comment by stobbsobe** [comment]
Thanks jmccrae, that makes sense. Please could that be implemented?

**Comment by jmccrae** [comment]
Implemented as definition change. I will merge in a few days if there are no further comments.
